### PR TITLE
add navigation ui test - PBS-282

### PR DESCRIPTION
**Pull request by bmorris-revacomm**
- added ui test for navigation (will need to update when we add more views)
- updated ui test for adding forms
- changed print button text to "Preview"

### CODE REVIEW

File: PBLogging/NewNavViews/Form781View/FlightsSection/SingleFlightRowView.swift [hunk 1]
```diff
@@ -13,6 +13,7 @@ struct SingleFlightRowView: View {
     
     var width:  CGFloat = 100
     var height: CGFloat = 40
+    
     var body: some View {
         VStack (alignment: .leading){
             HStack {
```
Comment (Pete-Hoch) [inline PBLogging/NewNavViews/Form781View/FlightsSection/SingleFlightRowView.swift:19]: I think you can revert this change and thus the whole file.

**Review by Pete-Hoch — APPROVED** [review]
This takes test coverage from 20% to almost 50%, fantastic!

Change in SingleFlightRowView.swift not needed.
Please reference PBS-282 in the title.

Fix, then squash & merge.

**Review by jbethanco — APPROVED** [review]
